hs_core/views/__init__.py

```diff
@@ -696,7 +696,7 @@ class AddUserForm(forms.Form):
                                       widget=autocomplete_light.ChoiceWidget("UserAutocomplete"))
 
     class AddGroupForm(forms.Form):
-        group = forms.ModelChoiceField(Group.objects.filter(gaccess__active=True).all(),
+        group = forms.ModelChoiceField(Group.objects.filter(gaccess__active=True).exclude(name='Hydroshare Author').all(),
```

Comment:
Agreed, but it is good to filter out this HydroShare Author group just to be sure for now.